internal/service/sagemaker/domain.go

```diff
@@ -389,6 +389,27 @@ func ResourceDomain() *schema.Resource {
 											ValidateFunc: verify.ValidARN,
 										},
 									},
+									"custom_image": {
```

Comment:
Hi @michaeljneely , I am unable to see expandDomainCustomImages under expandDomainCodeEditorAppSettings & flattenDomainCustomImages under flattenDomainCodeEditorAppSettings.

Reply:
Thanks, for catching this. Those missing elements have been added.

Reply:
LGTM